app/helpers/application_helper.rb

```diff
@@ -1,3 +1,12 @@
 # Comment
 module ApplicationHelper
+# Return the fulltitle on a per-paget title
+  def full_title(page_title = '')
```

Comment:
1. Xóa comment đi em
2. bỏ ngoặc của `full_title`
3. `page_title` chị có thể không khởi tạo giá trị ban đầu được ko em nhỉ?
4. phân biệt giúp chị empty?, nil? và blank? nào

Reply:
1. done.
2. done.
3. page_title có thể không cần khai báo trước.
4.  
- empty?: có thể được dùng cho 1 số object như là String, Hash, Array, nó sẽ chỉ trả về true khi các giá trị của nó là rỗng hoàn toàn.
- nil?: có thể được dùng cho tất cả các đối tượng, sẽ chỉ trả về true khi đối tượng là rỗng.
- blank? là 1 phương thức sẽ trả về true(tức được coi là blank) khi nó là nil, false, empty? hoặc là khoảng trắng.